Supplement ch5 (#9)
inbo/mbag-reports: source/mas_piloot_steekproefdesign/05_analyse_gegevens.Rmd

De volgende resultaten betreffen vogels.
Zoogdieren worden besproken in Sectie \@ref(exploratie-zoogdieren).
Binnen het pilootmeetnet worden al sinds 2018 tellingen verricht.

> \textcolor{red}{plan kiekendief ...} Zoals besproken in Sectie \@ref(piloot-overlap), zijn een aantal van deze punten hergebruikt bij de steekproeftrekking voor deze pilootstudie.

Author: Als er textcolor red staat, wacht ik nog op informatie van Johannes. Omdat ik ook nog niet weet wat hij al in eerdere hoofdstukken zal zetten.

**1. Regio**\
De regio's van het pilootmeetnet (De Moeren en Oostelijke leemstreek).

**2. Openheid landschap**\
In Sectie \@ref(piloot-stratificatie) werd reeds uitgelegd hoe per telpunt de openheid van het landschap werd berekend op schaal van 300 m.
Daarna werd deze continue variabele opgedeeld in verschillende klassen (OL: open landschap, HOL: halfopen landschap).

Naast deze categorische variabele berekenden we ook de openheid van het landschap als continue variabele op een schaal van 1000 m.
Deze schaal zou meer relevant zijn voor akkervogels \textcolor{red}{(REF)} en een continue variabele bevat meer informatie voor gedetailleerdere statistische analyses en het identificeren van subtiele patronen in de data.

**3. Soortbeschermingsplan**\
Categorische variabele binnen of buiten soortbeschermingsplan akkervogels (SBP).

> \textcolor{red}{Zowel binnen kerngebied als zoekzones zijn er soortbeschermende maatregelen (SB BO’s) gelegd (zie kaart VLM BO’s). In theorie waren deze maatregelen vooral bedoeld voor kerngebied, en zal daar een hogere dichtheid (bedekkingsgraad) liggen, maar de VLM heeft het toegelaten om ook in zoekzones deze maatregelen te sluiten (al was dat minder de prioriteit). Daar werd bv. in de Moeren geen rekening mee gehouden, en werden erg veel SB BO’s gelegd, ook omdat dit gebied wél kerngebied was van plan kiekendief (SBP Grauwe Kiekendief) waardoor er wel veel BO’s gesloten zijn. In de toekomst heeft VLM beloofd om meer/enkel rekening te houden met kerngebied. Voor het pilootmeetnet namen we de kerngebieden samen als onderzoeksgebied (binnen sbp) en alles hierbuiten beschouwen we als controle (buiten sbp).}

Reviewer: ```suggestion
\textcolor{red}{Zowel binnen kerngebied als zoekzones zijn er soortbeschermende maatregelen (SB BO’s) gelegd (zie kaart VLM BO’s). In theorie waren deze maatregelen vooral bedoeld voor kerngebied, en zal daar een hogere dichtheid (bedekkingsgraad) liggen, maar de VLM heeft het toegelaten om ook in zoekzones deze maatregelen te sluiten (al was dat minder de prioriteit). Daar werd bv. in de Moeren geen rekening mee gehouden, en werden erg veel SB BO’s gelegd, ook omdat dit gebied wél kerngebied was van plan kiekendief (SBP Grauwe Kiekendief) waardoor er wel veel BO’s gesloten zijn. In de toekomst wil men meer rekening houden met kerngebied. Voor het pilootmeetnet namen we de kerngebieden samen als onderzoeksgebied (binnen sbp) en alles hierbuiten beschouwen we als controle (buiten sbp).}
```

**4. Status teller**\
Analyses van telpunten die zowel door vrijwilligers als professionele teller zijn geteld, gaven duidelijk aan dat vrijwilligers een significant lager aantal waarnemingen en soorten zien dan een professionele teller (Sectie \@ref(dubbeltellingen)).
Voor elk telpunt hebben we dus ook nagegaan of er geteld is door een professionele teller of een vrijwilliger.

**5. Oppervlakte-aandeel kleine landschapselementen**\
Het aandeel oppervlakte van elke subklasse kleine landschapselementen binnen de telcirkels werd berekend via landschapselementen op of aangrenzend aan landbouwgebruikspercelen in Vlaanderen (onderzoek 2017 op basis van gegevens 2015).
De kaartlaag is vrij verkrijgbaar via @landschapselementen.
De verschillende subklasses zijn: (1) bomengroep, (2) bomenrij, (3) boom, (4) bosrand, (5) haag, (6) houtkant.

> **6. Perceelsgrootte (mediaan, IQR en CV)**\

Reviewer: CV is hier niet helemaal volgens definitie van CV. Misschien IQR / mediaan definiëren als "robuuste CV".

Ook zien we een verschillende trend over de jaren.
Om deze beter te begrijpen, bekeken we de oppervlakte aan soortbeschermingsplannen binnen de volledige perimeter van de pilootstudie over de jaren (Fig. \@ref(fig:total-propsb)).
We zien voor elk stratum een duidelijke toename tot 2022 en dan een lichte afname naar 2023.

> Dit houdt in dat we de correlatie tussen het aantal/de aanwezigheid van bepaalde akkervogels en soortbeschermingsmaatregelen zowel via kunnen nagaan via de categorische variabele `sbp` (binnen of buiten SBP) of via de continue variabele `area_prop_sb` (oppervlakte soortbeschermingsmaatregelen).

Reviewer: ```suggestion
Dit houdt in dat we de correlatie tussen het aantal/de aanwezigheid van bepaalde akkervogels en soortbeschermingsmaatregelen zowel via de categorische variabele `sbp` (binnen of buiten SBP) als via de continue variabele `area_prop_sb` (oppervlakte soortbeschermingsmaatregelen) kunnen nagaan.
```

De eerste fase is het schatten van de detectiekans door het fitten van een detectiecurve (Sectie \@ref(res-distance-stratum)).
De tweede fase is de ruimtelijke modellering via een soepele *smoothing function*.
Tweefasemodellen hebben het nadeel dat de onzekerheid van de detectiefunctie en de ruimtelijke modellen op de juiste manier gecombineerd moeten worden [@miller2013].
Wij hebben hier de onzekerheid van het detectieproces genegeerd wat te smalle geloofwaardigheidsintervallen kan opleveren voor schattingen van de densiteit.
Bovendien zijn we voorlopig niet dieper ingegaan op de keuze voor de smoothing function $f()$.
Ook hier kunnen verschillende afwegingen gemaakt worden [@wood2017].
In dit stadium van de pilootstudie was het niet mogelijk dieper in te gaan op deze tekortkomingen.
Bij de uitbreiding van een MAS voor heel Vlaanderen, zal deze methode verder op punt gesteld worden.

Indien we subgebieden willen afbakenen, bijvoorbeeld om interessante kernzones te identificeren, kunnen we gridcellen clusteren en daarna de predictie van de cluster vergelijken met een vooraf afgesproken waarde (bv., Is het aantal broedparen per 100 ha significant meer dan 10?).

> Hierbij moeten we een een methode vinden waarbij we op correcte wijze de onzekerheid berekenen op clusterniveau [@wadoux2023].

Reviewer: ```suggestion
Hierbij moeten we een methode vinden waarbij we op correcte wijze de onzekerheid berekenen op clusterniveau [@wadoux2023].
```

Een eerste mogelijkheid is om een variogram op te stellen o.b.v. de fitted values om een idee te krijgen van spatiale autocorrelatie.
Deze info kan dan gebruikt worden om de onzekerheid op clusterniveau te bepalen.
Een tweede mogelijkheid is om andere methodes en R packages te bekijken die dergelijke berekeningen integraal uitvoeren.
Ook deze mogelijkheden zullen in een later stadium onderzocht moeten worden.

> ### Correlaties met verklarende variabelen {#beheerovereenkomsten}

Reviewer: Ik zou in heel deze sectie niet spreken over correlaties. Correlaties zijn relaties tussen twee variabelen, terwijl we hier regressiemodellen fitten met 1 afh var en meerdere onafh vars. De titel kan misschien iets als "modellering in functie van verklarende variabelen" worden.

```

Analyses werden voor beide regio's apart gedaan via Bayesiaanse modellen met de **brms** package.
Selectie van variabelen werd gedaan via *projection predictive variable selection* methode (*PPVS*) [@piironen2017; @piironen2020; @catalina2022; @mclatchie2023].
Het doel van *PPVS* is het vinden van een zo klein mogelijke subset van kandidaat-variabelen met een voorspellende kracht die zo dicht mogelijk ligt bij die van het referentiemodel.
Als referentiemodel specificeerden we voor elke regio een Poisson model met alle variabelen in kwestie en is het best presterende model (in termen van voorspellende prestaties) dat we tot onze beschikking hebben.

> Voor de regressiecoëfficienten ($\beta$-parameters) gebruikten we de R2-D2 *shrinkage prior* [@zhang2016; @yanchenko2021].

Reviewer: Ter info: er is ondertussen ook deze https://arxiv.org/abs/2401.10180

Voor de hyperparameters gebruiken we 0,5 voor het gemiddelde van de Beta prior op $R^2$, 2 voor de precisie van de Beta prior en 1 voor de concentratievector van de Dirichlet prior op de variantie decompositieparameter (lagere waardes beteken een sterkere shrinkage).
Deze werden bepaald o.b.v. exploratieve analyses en bijlage B in @mclatchie2023.

We bemonsterden de a-posterioriverdelingen voor de modellen met drie *Markov chain Monte Carlo* (*MCMC*) ketens, die elk 8000 iteraties draaiden, waarvan de eerste 2000 werden verwijderd als *burn-in*.
De ketens werden verdund met een factor 3 zodat 2000 *posterior samples* per keten overbleven en dus 6000 *posterior samples* in totaal.
We onderzochten *MCMC*-convergentie door visuele beoordeling van traceplots en posterior densiteiten.
We onderzochten ook de *potential scale reduction factors* voor elke scalaire parameter in het model.
Ze lagen allemaal dicht bij 1, zoals vereist.
De model fit werd bevestigd op basis van *posterior predictive check* (*PPC*) waarbij we de geobserveerde uitkomst vergeleken met gesimuleerde datasets van de *posterior predictive distribution*.

> Na het fitten van de referentiemodellen voor elke regio *PPVS* toegepast met de **projpred** package [@R-projpred].

Reviewer: ```suggestion
Na het fitten van de referentiemodellen voor elke regio werd *PPVS* toegepast met de **projpred** package [@R-projpred].
```

De cumulatieve ranking proporties worden opgeteld naarmate submodelgrootte toeneemt.
Een cumulatieve ranking proportie voor variabele $x$ van 90 % bij $\leq$ 5 ($= j$) wil bijvoorbeeld zeggen dat $x$ in 90 % van de *cross-validation folds* folds een ranking kleiner of gelijk aan 5 heeft.

Finale modellen werden met de geselecteerde variabelen opnieuw gefit via **brms**.
We fitten zowel een Poisson als zero-inflated Poisson model.
Beide modellen werden vergeleken visueel via *PPCs* en o.b.v. het verschil in *ELPD* berekend via *5-fold cross-validation*.
Voor de nabewerking en visualisatie van de resultaten hebben we gebruik gemaakt van de functionaliteiten uit de **tidybayes** package [@R-tidybayes].
We rapporteren steeds de posterior mediaan van $exp(\beta_j) - 1$ als maat voor het centrale tendens van variabele $x_j$, en de standaard deviatie (= standaardfout) en 95 % geloofwaardigheidsinterval als maat voor onzekerheid.
Samenvattende statistieken van de verklarende variabelen zijn gegeven in Bijlage \@ref(mean-sd-contvars).

> Aangezien we geen interacties beschouwen is de trends gelijk voor alle categorieën van categorische variabelen.

Reviewer: ```suggestion
Aangezien we geen interacties beschouwen zijn de trends gelijk voor alle categorieën van categorische variabelen.
```

We definieerden een positief verschil als de ondergrens van het 95 % geloofwaardigheidsinterval groter is dan nul, een negatief verschil als de bovengrens kleiner is dan nul, een stabiel (geen) verschil als het geloofwaardigheidsinterval nul bevat maar binnen de referentiewaarden valt (tussen toename van 25 % en afname van 20 %) en een onzeker verschil als het geloofwaardigheidsinterval nul en ten minste een van de referentiewaarden bevat [@onkelinx2023].

#### Resultaten correlaties De Moeren

De voorspellende kracht van het model voor De Moeren bereikte reeds een maximum bij twee variabelen (Fig. \@ref(fig:ranking-dm-veldleeuw)).
Naast de vaste `area_prop_sb_cat` hebben we nog de variabele `boom` die in 100 % van de *cross-validation folds* aanwezig was (Fig. \@ref(fig:cvprop-dm-veldleeuw)).
Samenvattende statistieken van de parameters zijn gegeven in Tabel \@ref(tab:dm-veldleeuw-params).
De referentiegroep is nulbeleid (= geen oppervlakte‐aandeel soortbeschermingsmaatregelen aanwezig).

> We zien enkel een significant meer broedparen in telcirkels met hoogbeleid (gemiddeld gezien 119 % meer).

Reviewer: ```suggestion
We zien enkel significant meer broedparen in telcirkels met hoogbeleid (gemiddeld gezien 119 % meer).
```

Voor de andere categorieën is de onzekerheid te groot (Fig. \@ref(fig:beleid-dm-veldleeuw1)).
Dit komt omdat er voor sommige categorieën weinig data is.
We kunnen de categorieën laag- en middellaagbeleid aggregeren tot laagbeleid via een gewogen gemiddelde waarbij de gewichten afhankelijk zijn van het aantal data per groep.
Dit doen we ook voor hoogbeleid.
We zien een onzeker verschil in aantal broedparen voor laagbeleid en een positief verschil voor hoogbeleid met nulbeleid (gemiddeld gezien 109 % meer; Fig. \@ref(fig:beleid-dm-veldleeuw2)).

> Verder zien we een significante negatieve correlatie met de proportie bomen binnen de telcirkels (Fig. \@ref(fig:boom-dm-veldleeuw)).

Reviewer: ```suggestion
Verder zien we een significant negatief effect van de proportie bomen binnen de telcirkels (Fig. \@ref(fig:boom-dm-veldleeuw)).
```

                           "boom_veldleeuwerik_data.pdf"))
```

#### Resultaten correlaties Oostelijke leemstreek

De voorspellende kracht van het model voor de Oostelijke leemstreek neemt niet meer toe na negen variabelen (Fig. \@ref(fig:ranking-ol-veldleeuw)).
Naast de vaste `area_prop_sb_cat` hebben we nog de variabelen `openheid_waarde_1000`, `haag`, `houtkant`, `bomenrij`, `bosrand` en `boom` die in 100 % van de *cross-validation folds* aanwezig waren bij deze submodelgrootte en `meerjarige_fruitteelt_hfdtlt` en `grasland_hfdtlt_pre` in 80 % (Fig. \@ref(fig:cvprop-ol-veldleeuw)).
Samenvattende statistieken van de parameters zijn gegeven in Tabel \@ref(tab:ol-veldleeuw-params).
De referentiegroep is nulbeleid (= geen oppervlakte‐aandeel soortbeschermingsmaatregelen aanwezig).
We zien significant meer broedparen in telcirkels met laag-, middelhoog- en hoogbeleid (resp. gemiddeld 16 %, 49 % en 43 % meer).
Voor middellaagbeleid is de onzekerheid te groot (Fig. \@ref(fig:beleid-ol-veldleeuw1)).
Wanneer we de categorieën aggregeren zien we een positief verschil in aantal broedparen voor laagbeleid en hoogbeleid met nulbeleid Fig.
\@ref(fig:beleid-ol-veldleeuw2)).

> Verder zien we een significante positieve correlatie met de openheid van het landschap op 1000 m schaal (Fig. \@ref(fig:openheid-ol-veldleeuw)) en negatieve correlatie met de andere continue variabelen (Bijlage \@ref(extra-figs-2022-2023)).

Reviewer: ```suggestion
Verder zien we een significant positief effect van de openheid van het landschap op 1000 m schaal (Fig. \@ref(fig:openheid-ol-veldleeuw)) en negatieve effecten van de andere continue variabelen (Bijlage \@ref(extra-figs-2022-2023)).
```

include_graphics(file.path("media", "beheerovereenkomsten",
                           "oostelijke_leemstreek",
                           "beleidstoestand_veldleeuwerik_coarse.pdf"))
```

```{r openheid-ol-veldleeuw, out.width = "100%", fig.cap = "Relatie tussen het maximum aantal broedparen in een telcirkel en de openheid van het landschap op 1000 m schaal (grotere waardes betekenen een meer open landschap) waarbij de andere variabelen constant zijn gehouden op hun gemiddelde per categorie. De zwarte lijn is de posterior mediaan en de onzekerheidsband toont het 95 \\% geloofwaardigheidsinterval. Voor betere visualisatie hebben we een pseudo-log-transformatie langs de y-as gedaan."}
include_graphics(file.path("media", "beheerovereenkomsten",
                           "oostelijke_leemstreek",
                           "openheid_waarde_1000_veldleeuwerik_data.pdf"))
```

#### Discussie correlaties

Zowel in De Moeren als in de Oostelijke leemstreek zien we een groter maximum aantal broedparen per telcirkel als de proportie soortbeschermingsmaatregelen groter is.
Dit verschil is onzeker in klasses met weinig data.

> We moeten hierbij benadrukken dat dit een puur correlatief en géén causaal verband is.

Reviewer: ```suggestion
We moeten hierbij benadrukken dat deze effecten niet noodzakelijk wijzen op een causaal verband. Het betreft hier een immers observationele data met  belangrijke tekortkomingen om geobserveerde effecten te kunnen attribueren aan oorzakelijke verbanden.
```

> De soortbeschermingsmaatregelen zijn gekozen op plaatsen die reeds gunstig waren voor akkervogels [@agentschapvoornatuurenbos2021].

Reviewer: ```suggestion
De soortbeschermingsmaatregelen zijn immers gekozen op plaatsen die reeds gunstig waren voor akkervogels [@agentschapvoornatuurenbos2021].
```

> Deze analyses tonen de toestand van Veldleeuwerik in verschillende klasses van soortbeschermingsmaatregelen, geen trend.

Reviewer: ```suggestion
Bovendoen, tonen deze analyses de toestand van Veldleeuwerik in verschillende klasses van soortbeschermingsmaatregelen, geen trend.
```

> De andere variabelen die werden geselecteerd o.b.v. *PPVS* hadden allemaal een significante effect op het aantal broedparen in beide regio's.

Reviewer: ```suggestion
De andere variabelen die werden geselecteerd o.b.v. *PPVS* hadden allemaal een significant effect op het aantal broedparen in beide regio's.
```

De grootste verbeteringen die hier van toepassing zijn, zijn eerder gerelateerd aan de aggregatie van de data.
In deze analyses gebruikten we het maximum aantal broedvogels per telcirkel per jaar als responsvariabele.
Dit wil zeggen dat we geen rekening houden met de telperiodes, noch de exacte locatie waar een individu is waargenomen binnen de telcirkel.
We zijn er dan ook van overtuigd dat verdere analyses nog meer uit de data kan halen.
Enerzijds kunnen variabelen, zoals de aanwezigheid van hoofdteelten, veranderen over het jaar heen, wat de inclusie van tellingen per telperiode interessant maakt.
Anderzijds kunnen we aan elke specifiek observatie variabelen hangen.
Zo kan een vogel in een bepaald perceel met een bepaalde hoofdteelt worden geobserveerd waarbij er zich ook andere percelen met andere teelten in de telcirkel bevinden.
Nu aggregeren we de informatie van al die percelen op telcirkelniveau waarbij we deze informatie koppelen aan die observatie, zonder de exacte locatie van de observatie zelf te bekijken.
Minder aggregeren betekend meer data waardoor we meer subtiele verbanden kunnen waarnemen.

> Dit moeten in evenwicht zijn met de complexiteit van de modellen en de tijd die nodig is om deze te fitten.

Reviewer: ```suggestion
Dit moet echter in evenwicht zijn met de complexiteit van de modellen en de tijd die nodig is om deze te fitten.
```
